Makefile

```diff
@@ -67,6 +68,7 @@ test_integration: setup
 	$(MAKE) create_networks
 	-${DOCKER_COMPOSE_INFRA} up -d
 	${DOCKER_COMPOSE} up -d
+	while [ `docker ps | grep healthy | wc -l` != $(NUM_CONTAINERS) ] ; do sleep 1 ; echo -n .;done
```

Comment:
Do we think we need a timeout on this?

Reply:
Theres a util in .scripts that does this with a timeout arg if I remember correctly

Reply:
Yeah - thanks - its seems there is a remnant of it in the Makefile ... let me re-do this :)

Reply:
done